framework/wazuh/wdb.py

```diff
@@ -120,8 +120,14 @@ def execute(self, query, count=False):
             response = []
             step = limit if limit < self.request_slice and limit > 0  else self.request_slice
             for off in range(offset, limit+offset, step):
-                request = "{} limit {} offset {}".format(query_lower, step, off)
-                response.extend(self.__send(request))
+                try:
+                    request = "{} limit {} offset {}".format(query_lower, step, off)
+                    response.extend(self.__send(request))
+                except ValueError:
+                    request = "{} limit {} offset {}".format(query_lower, step//2, off)
```

Comment:
I've added an exception handler to prevent "incomplete JSON errors". If a JSON comes incomplete from Wazuh DB, the half of items is requested.